src/main/java/com/endava/soa_5/JUnit_and_Hamcrest/MagicSquare.java

```diff
@@ -0,0 +1,111 @@
+package com.endava.soa_5.JUnit_and_Hamcrest;
+
+public class MagicSquare {
+
+    private int[][] intArray;
+
+    private int constant;
+
+    public MagicSquare(int[][] intArray) {
+
+        this.intArray = intArray;
+        constant = intArray.length * (intArray.length * intArray.length + 1) / 2;
+    }
+
+    public boolean isNotEmpty() {
+        if (intArray.length == 0) {
+            return false;
+        } else {
```

Comment:
else can be removed. Just return true after the if statement code block

Reply:
Done!